meilisearch-release-changelog/v1.11.0.md

```diff
@@ -0,0 +1,182 @@
+* The new document template changed to
+  * filter out fields that are not searchable
+  * filter out fields that are nil or missing for the current document
+
+v1.10 and before:
+```
+{% for field in fields %}
+{{ field.name }}: {{ field.value }}\n
+{% endfor %}
+```
+
+From v1.11:
+```
+{% for field in fields %}
+{% if field.is_searchable and not field.value == nil %}
+{{ field.name }}: {{ field.value }}\n
+{% endif %}
+{% endfor %}
+```
+
+This last change is backward compatible. Embedders that use the v1.10 document template will keep using it. The new default document template will only be applied to newly added embedders.
+
+## Speed up indexing of your vector database with binary quantization
+
+TBD
+
+## Federated search improvements
+
+### Request by-index facet distribution and stats
+
+Request facet distributions and facet stats in your federated search, by using `federation.facetsByIndex` in the POST `POST /multi-search` route.
+
+By default the distribution and stats are returned for each index that participates in the federated search.
+
+```json5
+POST /multi-search
+{
+  "federation": {
+    "limit": 20,
+    "offset": 0,
+	"facetsByIndex": {
+	  "movies": ["title", "id"],
+	  "comics": ["title"],
+	}
+  },
+  "queries": [
+    {
+      "q": "Batman",
+      "indexUid": "movies"
+    },
+    {
+      "q": "Batman",
+      "indexUid": "comics"
+    }
+  ]
+}
+```
+
+### Request merged facet distribution and stats
+
+Merge the returned facets in a single facet distribution and stats that is global to the entire request, by using `federation.mergeFacets` in the POST `POST /multi-search` route.
+
+The number of returned values can be limited for all facets with `federation.mergeFacets.maxValuesPerFacet`.
+
+```json5
+POST /multi-search
+{
+
+  "federation": {
+    "limit": 20,
+    "offset": 0,
+	  "facetsByIndex": {
+	    "movies": ["title", "id"],
+	    "comics": ["title"],
+	  },
+	  "mergeFacets": {
+	    "maxValuesPerFacet": 10,
+	  }
+  }
+  "queries": [
+    {
+      "q": "Batman",
+      "indexUid": "movies"
+    },
+    {
+      "q": "Batman",
+      "indexUid": "comics"
+    }
+  ]
+}
+```
+
+Done by @dureuill in #4929.
+
+## Experimental - `START_WITH` operator during search
+
+Enable the experimental feature to use the `START_WITH` filter operator:
+
+```bash
+curl \
+  -X PATCH 'http://localhost:7700/experimental-features/' \
+  -H 'Content-Type: application/json' \
+  --data-binary '{
+    "containsFilter": true
+  }'
+```
+
+Use the `START_WITH` operator when filtering:
+
+```json5
+curl \
+  -X POST http://localhost:7700/indexes/movies/search \
+  -H 'Content-Type: application/json' \
+  --data-binary '{
+    "filter": "hero START_WITH spider"
+  }'
+```
+
+🗣️ This is an experimental feature, and we need your help to improve it! Share your thoughts and feedback on this [GitHub discussion](https://github.com/orgs/meilisearch/discussions/763).
```

Comment:
(I tried editing the text but there are too many backticks, GitHub is not happy)

Can you change all `START_WITH` to `STARTS_WITH`, please? 😃

Reply:
🙈